node-ui/src/api/dataSource/NodeDataSource.ts

```diff
@@ -198,6 +212,20 @@ export namespace WalletType {
   }): WalletType {
     return { type: 'STARKNET', walletName } as SNWalletType;
   }
+
+  export function INTERNETCOMPUTER({
+    cannisterId = 'rdmx6-jaaaa-aaaaa-aaadq-cai',
```

Comment:
What is this value rdmx6-jaaaa-aaaaa-aaadq-cai' ?

Reply:
this is the default value for the canister ID. There are two canisters that are used, one for production (mainnet) and one for staging (testnet) similar to the NEAR network type.

Reply:
Ok, can u please extract it to const to indicate what it is